packages/daemon/src/types.d.ts

```diff
@@ -444,20 +444,8 @@ export interface Mail {
     edgeNames: Array<string>,
     petNames: Array<string>,
   ): Promise<void>;
-  respond(
-    what: string,
-    responseName: string,
-    senderId: string,
-    senderPetStore: PetStore,
-    recipientId?: string,
-  ): Promise<unknown>;
-  receive(
-    senderId: string,
-    strings: Array<string>,
-    edgeNames: Array<string>,
-    ids: Array<string>,
-    receiverId: string,
-  ): void;
+  // TODO attenuate
+  deliver(message: EnvelopedMessage): void;
```

Comment:
Attenuate in what sense? I'm not sure what this means in the context of the type definition.

Reply:
Left over. I’ll delete the comment.